modules/eks/cert-manager/variables.tf

```diff
@@ -112,13 +124,14 @@ variable "cert_manager_issuer_values" {
 
 variable "create_namespace" {
   type        = bool
-  description = "Create the namespace if it does not yet exist. Defaults to `false`."
-  default     = null
+  description = "Create the namespace if it does not yet exist. Defaults to `true`."
+  default     = true
```

Comment:
👍  since we now have a k8 resource tracking it in our module, can we apply this change to all variables.tf helm components in this PR (`alb-controller`, and the rest) and set it as the default way moving forward

Reply:
No. It is not uncommon to deploy external-dns and alb-controller to `kube-system` and we do not want to try to create that namespace. I only set it to true for `cert-manager` because it is a recommended security practice to keep `cert-manager` in an isolated namespace.